tornado/ioloop.py

```diff
@@ -257,7 +258,9 @@ def current(instance: bool = True) -> Optional["IOLoop"]:  # noqa: F811
         except RuntimeError:
             if not instance:
                 return None
-            # Create a new asyncio event loop for this thread.
+            if threading.current_thread() is not threading.main_thread():
```

Comment:
I don't think we want the main-thread-only behavior here. Tornado has historically allowed event loop creation on any thread (that's what AnyThreadEventLoopPolicy is for), so it's a nice side effect that we can bring back the "any thread" behavior by default at the same time that AnyThreadEventLoopPolicy is being deprecated. 

Just remove this commit.

Reply:
OK, I've removed that commit, and adjusted `AnyThreadEventLoopPolicyTest` to match.